tests/unit/channel_test.py

```diff
@@ -33,6 +35,34 @@ def ssh_channel(ssh_client_session):
         chan.close()
 
 
+def test_open_session_timeout(ssh_session_connect):
+    """Test opening a new channel with a low timeout value.
+
+    This generates an exception from ssh_channel_open_session()
+    returning SSH_AGAIN with the usec timeout and default
+    open_session_retries value of 0.
+    """
+    ssh_session = Session()
```

Comment:
It doesn't seem like the session gets closed in here.

Reply:
Added the .close()